okapi/core.php

```diff
@@ -1206,6 +1206,83 @@ public static function get_node_code()
     }
 
     /**
+     * Return the recommended okapi_base_url.
+     *
+     * This is the URL which we want all *new* client applications to use.
+     * OKAPI will suggest URLs with this prefix in various context, e.g. in all
+     * the dynamically generated docs.
+     *
+     * Also see `get_allowed_base_urls` method.
+     */
+    public static function get_recommended_base_url()
+    {
+        return Settings::get('SITE_URL')."okapi/";
```

Comment:
See #343, postings 3, 4, 7, 9, 10. Unfortunately, #343 was not done before https was enabled at www.opencaching.de, and I forgot to disable https://www.opencaching.de/okapi. Therefore OKAPI-https now is in use, though it was not cleanly designed.